extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcUtils.java

```diff
@@ -16,14 +18,36 @@ private OidcUtils() {
 
     }
 
+    public static boolean validateClaims(OidcConfig.Token tokenConfig, JsonObject json) {
+        if (tokenConfig.issuer.isPresent()) {
+            String issuer = json.getString(Claims.iss.name());
+            if (!tokenConfig.issuer.get().equals(issuer)) {
+                throw new OIDCException("Invalid issuer");
+            }
+        }
+        if (tokenConfig.audience.isPresent()) {
+            Object claimValue = json.getValue(Claims.aud.name());
+            List<String> audience = Collections.emptyList();
+            if (claimValue instanceof JsonArray) {
+                audience = ((JsonArray) claimValue).stream().map(v -> v.toString()).collect(Collectors.toList());
```

Comment:
We don't allow lambda in runtime code, they use up a significant amount of additional memory.

Reply:
done